org.eclipse.jdt.core/compiler/org/eclipse/jdt/internal/compiler/lookup/BlockScope.java

```diff
@@ -851,6 +852,16 @@ public VariableBinding[] getEmulationPath(LocalVariableBinding outerLocalVariabl
 	return null;
 }
 
+private Object[] getSyntheticEnclosingArgumentLambda(MethodScope methodScope, ReferenceBinding targetEnclosingType) {
```

Comment:
Trying to parse the method name as an English sentence, I struggle to find a syntax rule for token 'Lambda' :)

IOW: it's hard to see the connection between the synth arg and the lambda. Perhaps just a tiny 'of' will clarify, if that's what it is. But then s.o. will be puzzled: how is the lambda represented in the signature ...

Apparently, the call site uses the abbrev. EAL derived from this method name, so when updating the method name, one should also have a look at its caller.